modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -2189,4 +2189,10 @@
     <string name="referrals_create_subscription_failed">Could not create subscription. Please try again later.</string>
     <string name="referrals_redeem_code_failed">Could not redeem code. Please try again later.</string>
 
+    <!-- Manage Downloaded Episodes -->
+    <string name="pencil_clean_up_icon_content_description">Pencil Clean Up</string>
+    <string name="you_are_running_low_on_storage">You\'re running low on storage</string>
+    <string name="save_space_by_managing_downloaded_episodes">Save %1$s – by managing downloaded episodes.</string>
```

Comment:
The copy was updated after putting this PR to match with this request: p1729915301848409/1729891135.765949-slack-C07TL9YK9V1

That's why the video is showing a different copy